tests/integration_tests/performance/test_block_performance.py

```diff
@@ -172,8 +172,8 @@ def read_values(cons, numjobs, env_id, mode, bs, measurement, logs_path):
             f"{logs_path}/{env_id}/{mode}{bs}/{mode}"
             f"{bs}_{measurement}.{job_id + 1}.log"
         )
-        file = open(file_path, encoding="utf-8")
-        lines = file.readlines()
+        with open(file_path, encoding="utf-8") as file:
+            lines = file.readlines()
```

Comment:
How about `file_path.read_text(encoding("utf-8").splitlines()`

Reply:
ohh, nice, didn't know about that one! is there something similar we could use for the json files?

Reply:
I do `json.loads(file_path.read_text(encoding...))`

Reply:
Alright, github doesnt display your comment for some reason, but I did the `json.loads(path.read_text())` thingy anyway